numpy/core/include/numpy/ufuncobject.h

```diff
@@ -209,9 +228,21 @@ typedef struct _tagPyUFuncObject {
          * set by nditer object.
          */
         npy_uint32 iter_flags;
+
+        ufunc_extension s_extension;
```

Comment:
This can't go here, as it changes the size of `PyUfunc_Object` which is an ABI change. I think we're forced to allocate it on the heap.

Reply:
Hmm, actually that suggestion doesn't help